.build/tasks/generateHelp.PlatyPS.build.ps1

```diff
@@ -41,26 +48,162 @@ Task UpdateHelp {
+    "`tProject Name             = '$ProjectName'"
+    "`tOutput Directory         = '$OutputDirectory'"
+    "`tPester Output Folder     = '$PesterOutputFolder"
+
+    if ($VersionedOutputDirectory)
+    {
+        # VersionedOutputDirectory is not [bool]'' nor $false nor [bool]$null
+        # Assume true, wherever it was set
+        $VersionedOutputDirectory = $true
+    }
+    else
     {
-        $BuildOutput = Join-Path -Path $ProjectPath.FullName -ChildPath $BuildOutput
+        # VersionedOutputDirectory may be [bool]'' but we can't tell where it's
+        # coming from, so assume the build info (Build.yaml) is right
+        $VersionedOutputDirectory = $BuildInfo['VersionedOutputDirectory']
+    }
+
+    $GetBuiltModuleManifestParams = @{
+        OutputDirectory          = $OutputDirectory
+        BuiltModuleSubDirectory  = $BuiltModuleSubDirectory
+        ModuleName               = $ProjectName
+        VersionedOutputDirectory = $VersionedOutputDirectory
+        ErrorAction              = 'Stop'
     }
 
-    if (-not (Split-Path -Path $HelpFolder -IsAbsolute))
+    $builtModuleBase = Get-SamplerBuiltModuleBase @GetBuiltModuleManifestParams
+    "`tBuilt Module Base        = '$builtModuleBase'"
+
+    $builtModuleManifest = Get-SamplerBuiltModuleManifest @GetBuiltModuleManifestParams
+    "`tBuilt Module Manifest    = '$builtModuleManifest'"
+
+    if ($builtModuleRootScriptPath = Get-SamplerModuleRootPath -ModuleManifestPath $builtModuleManifest)
     {
-        $HelpFolder = Join-Path -Path $SourcePath -ChildPath $HelpFolder
+        $builtModuleRootScriptPath = (Get-Item -Path $builtModuleRootScriptPath -ErrorAction SilentlyContinue).FullName
     }
 
-    Import-Module -Name $ProjectName -Force
-    Update-MarkdownHelpModule -Path $HelpFolder
+    "`tBuilt ModuleRoot script  = '$builtModuleRootScriptPath'"
+
+    $ModuleVersion = Get-BuiltModuleVersion @GetBuiltModuleManifestParams
+    $ModuleVersionObject = Split-ModuleVersion -ModuleVersion $ModuleVersion
+    $ModuleVersionFolder = $ModuleVersionObject.Version
+    $preReleaseTag       = $ModuleVersionObject.PreReleaseString
+
+    "`tModule Version           = '$ModuleVersion'"
+    "`tModule Version Folder    = '$ModuleVersionFolder'"
+    "`tPre-release Tag          = '$preReleaseTag'"
+
+    $HelpFolder = Get-SamplerAbsolutePath -Path $HelpFolder -RelativeTo $SourcePath
+    "`tHelp Folder Path         = '$HelpFolder'"
+
+    $HelpOutputFolder =  Get-SamplerAbsolutePath -Path $HelpOutputFolder -RelativeTo $OutputDirectory
+    "`tHelp output Folder        = '$HelpOutputFolder'"
+
+    $HelpOutputVersionFolder = Get-SamplerAbsolutePath -Path $ModuleVersion -RelativeTo $HelpOutputFolder
+    "`tHelp output Version Folder= '$HelpOutputVersionFolder'"
+
+    $HelpOutputCultureFolder = Get-SamplerAbsolutePath -Path $HelpCultureInfo -RelativeTo $HelpOutputVersionFolder
+    "`tHelp output Culture path  = '$HelpOutputCultureFolder'"
```

Comment:
This assumes the docs are relative to the module source, not the project (BuildRoot). Most of what I have seen from other projects is that the MD docs are in the root of the project.

Reply:
hmmmmm. Good question, @johlju what do you think?
Technically you could override the default (when I've implemented the config), using, say. `../docs`.

Reply:
I'd add that maybe we should have an option to build the MAML into the built module base's CultureInfo folder.